Start.tsx

```diff
@@ -187,11 +205,11 @@ function Start() {
       <ErrorBoundary onError={logError} FallbackComponent={MemmyErrorView}>
         {/* eslint-disable-next-line react/style-prop-object */}
         <StatusBar style={statusBarColor} />
-        <GestureHandlerRootView style={{ flex: 1 }}>
+        <GestureHandlerRootView style={{ flex: 1, backgroundColor: selectedTheme.colors.app.bg }}>
```

Comment:
This color doesn't seem to leak anywhere except for during launch right? So what if we instead of using the theme's background color set this to the background color of the splashscreen? For reference that color is `#292524`

Reply:
Here are the recordings using that color.

**Onboarding**
![Simulator Screen Recording - iPhone 14 Pro Max - 2023-07-13 at 22 52 29](https://github.com/Memmy-App/memmy/assets/3818037/c92b6198-b585-4061-8351-66d18430c36a)
**Logged In**
![Simulator Screen Recording - iPhone 14 Pro Max - 2023-07-13 at 22 54 58](https://github.com/Memmy-App/memmy/assets/3818037/b779cdec-2464-40bf-838e-34c52ff3e19e)

Vs. The theme background color.

**Onboarding**
![Simulator Screen Recording - iPhone 14 Pro Max - 2023-07-13 at 22 58 58](https://github.com/Memmy-App/memmy/assets/3818037/3998efe2-cabd-4295-a32b-db84dffd0dea)
**Lo
![Simulator Screen Recording - iPhone 14 Pro Max - 2023-07-13 at 22 59 54](https://github.com/Memmy-App/memmy/assets/3818037/4f554ca1-d5bc-4705-8905-99771f688131)
gged In**


To me it seems like the theme background color is less obvious.

Reply:
Oh yep, I think you're right. It actually does a flash back and forth when using the splash BG color.

I think we should stick with what you have then and we can see about that onboarding flash. If you think of a solution off the top of your head go for it but I don't think it's a major issue at the moment and removing the flash from the normal use is the biggest priority.

Reply:
I don't have any quick ideas for that since we'd need to go down to the stack level to know what the user is looking at. We can probably eventually figure something out, but I'm thinking it's going to be more trouble than it's worth in the short term. Right now Memmy is in "broad brush strokes" territory, not "polishing light switches and door knobs" territory.